recipes/r-ggalluvial/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = '0.9.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-ggalluvial
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/ggalluvial_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/ggalluvial/ggalluvial_{{ version }}.tar.gz
+  sha256: 49ea3c5dec8b0627b65a749cf545c54b5a71a7ef19c8692988362b2db311ca34
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: true  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
+  host:
+    - r-base >=3.3.0
```

Comment:
The version restriction can be removed, the min R version conda-forge uses is 3.4.1. That should also make the package build on Windows (bug in conda-build.).